src/commands/generate_comments.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2022-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+
+const {FakeListChatModel} = require('@langchain/core/utils/testing');
+const {PromptTemplate} = require('@langchain/core/prompts');
+const {RunnableSequence} = require('@langchain/core/runnables');
+const {StringOutputParser} = require('@langchain/core/output_parsers');
+const {readFileSync, writeFileSync} = require('fs');
+const {BaseChatModel} = require('@langchain/core/language_models/chat_models');
+const { ChatOpenAI } = require('@langchain/openai');
+
+/**
+ * Construct LLM chat model based on provider specified in cli options
+ *
+ * @param {Object} opts - All options
+ * @return {BaseChatModel} Constructed model
+ */
+function makeModel(opts) {
+  switch (opts.provider) {
+  case 'placeholder':
```

Comment:
@Dirakon it seems something's wrong with indentation here

Reply:
`eslint` task requires the indentation to be like this. As can be seen in this failing pipeline - https://github.com/objectionary/eoc/actions/runs/14742035094/job/41382039395?pr=461

To change this behavior, we could add `{ "SwitchCase": 1 }` in `eslint.config.js` (like it is suggested [here](https://stackoverflow.com/a/53055584)).

Should I modify `eslint.config.js` file? I checked locally - this change would indeed fix indentation in this place (and it won't affect any other files).

Reply:
@Dirakon Let's do it

Reply:
I added the setting and fixed the indentation in the latest commit